code/modules/gear_presets/survivors/solaris/preset_solaris.dm

```diff
@@ -79,9 +79,25 @@
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/jackboots(new_human), WEAR_FEET)
 	..()
 
+/datum/equipment_preset/survivor/uscm/solaris
+	name = "Survivor - Solaris United States Colonial Marine Corps Recruiter"
+	assignment = "USCM Recruiter"
+	paygrade = PAY_SHORT_ME5
+
+/datum/equipment_preset/survivor/uscm/solaris/load_gear(mob/living/carbon/human/new_human)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/under/marine/officer/bridge(new_human), WEAR_BODY)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/accessory/ranks/marine/e5(new_human), WEAR_ACCESSORY)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/backpack/marine/satchel(new_human), WEAR_BACK)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/suit/storage/jacket/marine/service(new_human), WEAR_JACKET)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/belt/gun/m4a3/full(new_human), WEAR_WAIST)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/head/cmcap(new_human), WEAR_HEAD)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/marine/knife(new_human), WEAR_FEET)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/pouch/general/medium(new_human), WEAR_R_STORE)
+	..()
```

Comment:
Will this be an issue because it won't call `add_ice_colony_survivor_equipment()` until after adding all these things?

Reply:
It's on Solaris, does it need a ice colony survivor equipment because no other one in the preset does.